### PR TITLE
Add robots.txt

### PR DESCRIPTION
**Pull request by michelle-rago**
Add robots.txt

### CODE REVIEW

**Comment by michelle-rago** [comment]
@evoeges Can you review this PR? I need another review before I can merge it. I'm adding a basic robots.txt file. Robots.txt files aren't required anymore, but I'm adding in case it will support discoverability. I'm following search.gov recommendations for it.

**Review by evoeges — APPROVED** [review]

**Comment by michelle-rago** [comment]
@caleywoods I decided to add a robots.txt. I'm getting a 404 error on it though. (If this will take too much time to look into just let me know.) Do I need to add something to the eleventy.js file? The 18F guides has this reference in their .js file:

```
  // Copy the robots.txt file to the output
  config.addPassthroughCopy('robots.txt');
```

**Pull request merged**
